app/assets/v2/js/grants/detail.js

```diff
@@ -1,3 +1,33 @@
 $(document).ready(function() {
   $('#tabs').tabs();
+
+  $('#edit--title').on('click', () => {
+    inlineEdit('#form--input__title', '#edit--title');
+  });
+
+  $('#edit--description').on('click', () => {
+    inlineEdit('#form--input__description', '#edit--description');
+  });
 });
+
+const inlineEdit = (input, icon) => {
+  if ($(icon).hasClass('fa-edit'))
+    makeEditable(input, icon);
+  else
+    inlineSave(input, icon);
+};
+
+const makeEditable = (input, icon) => {
+  $(input).addClass('editable');
+  $(input).prop('readonly', false);
+  $(input).prop('disabled', false);
+  $(icon).toggleClass('fa-edit').toggleClass('fa-save');
+};
+
+const inlineSave = (input, icon) => {
+  $(input).removeClass('editable');
+  $(input).prop('readonly', true);
+  $(input).prop('disabled', true);
+  $(icon).toggleClass('fa-edit').toggleClass('fa-save');
+  // TODO : fire API
```

Comment:
@mbeacom

Reply:
@thelostone-mc You just want an endpoint to `POST` the edit to?

Reply:
yup